crates/ursa-index-provider/src/provider.rs

```diff
@@ -9,6 +9,7 @@ use axum::{
     response::{IntoResponse, Response},
 };
 use fvm_ipld_blockstore::Blockstore;
+#[allow(deprecated)]
```

Comment:
why we using the deprecated

Reply:
> allow depreciated use of the Cbor trait in ursa-index-provider, because ipld_encoding 0.3.3 causes warnings. We require our own implementation because the new one breaks integrations